ocs_ci/deployment/deployment.py

```diff
@@ -635,11 +635,6 @@ def deploy_ocs_via_operator(self, image=None):
                 and not lso_type == constants.AWS_EBS
             ):
                 deviceset_data["count"] = 2
-            if ocs_version >= 4.5:
```

Comment:
@lukebrowning you can add ```if ocs_version >= 4.5 and config.ENV_DATA["platform"] != constants.IBMCLOUD_PLATFORM``` we can move forward with this change faster, in general we dont need the if but until we get some tier1 failure sorted out we can have it for now.

Reply:
not needed, resolving.